include/hip/hip_ext.h

```diff
@@ -22,10 +22,8 @@ THE SOFTWARE.
 
 #ifndef HIP_INCLUDE_HIP_HIP_HCC_H
 #define HIP_INCLUDE_HIP_HIP_HCC_H
-
-#ifdef __HCC__
-
 #include "hip/hip_runtime_api.h"
+#ifdef __HCC__
```

Comment:
Please include this header, and also use #ifdef __HCC__ && __HIP_VDI__ for HIP-Clang support.

Reply:
but hip-clang doesn't `hc::accelerator` and `hc::accelerator_view`